controllers/comment-controllers.js

```diff
@@ -0,0 +1,43 @@
+const { deleteComment, addComment } = require("../models/comment-models");
+const {
+  checkCommentIdExists,
+  checkIdExists,
+  checkUserNameExist,
+} = require("../models/article-parameters-exists-models");
+
+const postComment = (request, response, next) => {
+  const { body } = request.body;
+  const { username } = request.body;
+  const { article_id } = request.params;
+  if (body === undefined || username === undefined) {
```

Comment:
If your error handling is set up correctly, you shouldn't need to do this as both body and username cannot be null in this comments table (which we're trying to post to). PSQL will throw a not null violation error

Reply:
Hi, I set the if statement up to respond with a custom message that explains that there is a missing field as opposed to incorrectly entered data which may still be valid. I'm not sure how I would set up error handling with the null violation to specifically explain this whilst separating it from an error thrown for incorrect syntax for example